platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -4840,14 +4840,7 @@ void DisplayServerX11::process_events() {
 
 					WindowID window_id_other = INVALID_WINDOW_ID;
 					Window wd_other_x11_window;
-					if (wd.focused) {
```

Comment:
This code was from #69678 but removing it fixes the hyperlinks in tooltip and from what I could test, I was not able to reproduce the problem with the Color picker on Ubuntu Wayland.